tests/apis/platformvm/api.test.ts

```diff
@@ -2381,9 +2387,9 @@ describe("PlatformVMAPI", (): void => {
       const txu1Outs: TransferableOutput[] = (
         txu1.getTransaction() as AddValidatorTx
       ).getOuts()
-      const txu1Stake: TransferableOutput[] = (
-        txu1.getTransaction() as AddValidatorTx
-      ).getStakeOuts()
+      //const txu1Stake: TransferableOutput[] = (
+      //  txu1.getTransaction() as AddValidatorTx
+      //).getStakeOuts()
```

Comment:
Do we need to leave that as commented in?

Reply:
Should remain to replace them with LockedIn / LockedOut logic as soon they are in.